volatility3/framework/plugins/mac/ifconfig.py

```diff
@@ -9,7 +9,7 @@
 
 
 class Ifconfig(plugins.PluginInterface):
-    """Lists loaded kernel modules"""
+    """ Lists network interface information for all devices """
```

Comment:
There's now a space at the start of this comment, might be best to just launch right into the comment?

Reply:
As I referred to the description of volatility 2, it seems to have been moved as it is. I modified it!